vaadin/spreadsheet
# chore: Updates for modern Eclipse compatibility with Java 11

- Add missing version numbers
- Update Vaadin, TestBench, Charts, and Jetty versions
- Update maven-compiler-plugin and maven-jar-plugin versions
- Merge duplicate plugin definitions
- Add Java 8 source configuration to maven-javadoc-plugin
- Add compiler source/target and sourceEncoding to all modules
- Ensure all maven-compiler-plugin definitions use Java 8 and UTF-8
- Add profile with maven-compiler-plugin for JDK 11 in the Charts module
